File: c/Lista2/Desafio1.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){float EstoqueMedio, QuantMin, QuantMax;

printf("Digite a quantidade mínima de peça: ");
scanf("%f", &QuantMin);

printf("Digite a quantidade máxima de peça: ");
scanf("%f", &QuantMax);

EstoqueMedio = (QuantMin + QuantMax) / 2;
printf("O estoque médio de peças é: %3.0f", EstoqueMedio);
    
}

File: js/lista2/Desafio1.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
}); 

/*
Desafio1- Faça um algoritmo para:

Calcular o estoque médio de uma peça;

Fórmula: ESTOQUE MÉDIO = (QUANTIDADE MÍNIMA + QUANTIDADE MÁXIMA) / 2.
*/

rl.question('Quantidade Máxima: ', (quantidadeMaxima) => {
    rl.question('Quantidade Mínima: ', (quantidadeMinima) => {
      let operacao = (quantidadeMinima + quantidadeMaxima);
      let estoqueMedio = operacao / 2;

        
        console.log(`O estoque médio de peças é: ${estoqueMedio}`);
      rl.close();
    });
  });

